sdl/sdl_gpu.h

```diff
@@ -0,0 +1,105 @@
+/**
+ * @file sdl_gpu.h
+ *
+ */
+
+#ifndef SDL_GPU_H
+#define SDL_GPU_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#ifndef LV_DRV_NO_CONF
+#ifdef LV_CONF_INCLUDE_SIMPLE
+#include "lv_drv_conf.h"
+#else
+#include "../../lv_drv_conf.h"
+#endif
+#endif
+
+#if USE_SDL_GPU
+
+#ifdef LV_LVGL_H_INCLUDE_SIMPLE
+#include "lvgl.h"
+#else
+#include "lvgl/lvgl.h"
+#endif
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+/**
+ * Initialize SDL to be used as display, mouse and mouse wheel drivers.
+ */
+void sdl_init(void);
+
```

Comment:
Please a comment to this  `sdl_gpu_disp_drv_init` functions and emphasize that they need to be used instead of LVGL's `lv_disp_draw_buf_init` and `lv_disp_drv_init`.

Reply:
Added a bit more description.

Reply:
Great thanks.